src/gui.c

```diff
@@ -70,6 +76,11 @@ window_init ();
 static void
 plug_init ();
 
+static void uzbl_commit_cb(
+  GtkIMContext* context,
+  const gchar* str,
+  UzblGui* gui);
```

Comment:
Current style would be:

```c
static void
uzbl_commit_cb(GtkIMContext *context, const gchar *str, gpointer data);
```

With a cast into the `UzblGui` type in the function. Also would be better named as `uzbl_input_commit_cb`.